app/models/pawn.rb

```diff
@@ -30,17 +32,27 @@ def one_forward_move?(x, y)
 
   # Returns true if coordinates provided are
   # 2 tiles forward from the piece's origin.
-  def two_forward_move?(x, y)
+  def clear_two_forward_move?(x, y)
     if color == 'black'
-      x_distance(x) == 0 && y == y_position + 2
+      x_distance(x) == 0 && y == y_position + 2 && path_clear?(x, y, 2)
     else
-      x_distance(x) == 0 && y == y_position - 2
+      x_distance(x) == 0 && y == y_position - 2 && path_clear?(x, y, 2)
     end
   end
 
   # Returns true if the coordinates provided
   # are 1-2 tiles forward from the piece's origin.
   def first_forward_move?(x, y)
-    one_forward_move?(x, y) || two_forward_move?(x, y)
+    one_forward_move?(x, y) || clear_two_forward_move?(x, y)
+  end
+
+  # Returns true if there is any piece present
+  # in the tile provided.  Will be refactored when
+  # merged with capture logic.
+  def forward_capture?(x, y)
```

Comment:
Had trouble with ABC size errors with embedding it into the the valid move helper methods and I am loathe to make the parent valid_move method less readable so I kept it in a helper for now.